sql/core.go

```diff
@@ -110,6 +110,10 @@ type Node interface {
 type Expressioner interface {
 	// Expressions returns the list of expressions contained by the node.
 	Expressions() []Expression
+	// TransformExprssions applies for each expression in this node
```

Comment:
s/TransformExprssions/TransformExpressions/

Reply:
Sounds like something that an intelligent code review bot should be able to detect, [hint](https://github.com/src-d/feature-idea/issues/new?template=Custom.md)